examples/particle/aws/kinesis/kinesis_ex.py

```diff
@@ -0,0 +1,35 @@
+from pcf.particle.aws.kinesis.kinesis import Kinesis
+from pcf.core import State
+import time
+
+example_kinesis_json = {
+    "pcf_name": "pcf_example",
+    "flavor": "kinesis",
+    "aws_resource": {
+        "StreamName": "pcfKineTest3",
+        "ShardCount": 8,
+        "RetentionPeriodHours": 168,
+        "EncryptionType": 'KMS',
+        "KeyId": 'alias/aws/kinesis',
+        "Tags": [{
+            "Key": "test1",
+            "Value": "test2"
+        }, {
+            "Key": "OwnerContact",
+            "Value": "aaron.gill@capitalone.com",
+            "Key": "ASV",
+            "Value": "ASVCLOUDCUSTODIAN"
```

Comment:
could you change all the key values to dummy key values

Reply:
:squidward_dab: